org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/xhtml/CDANarrativeFormat.java

```diff
@@ -257,6 +258,12 @@ private void processTr(Element e, XhtmlNode xn) throws FHIRException {
     processChildren(e, xc);
   }
 
+  private void processLinkHtml(Element e, XhtmlNode xn) throws FHIRException {
+    XhtmlNode xc = xn.addTag("linkHtml");
+    processAttributes(e, xc, "ID", "language", "styleCode", "align", "char", "charoff", "valign");
```

Comment:
This sequence of attributes is repeated a few times. A good descriptive constant would be useful.

Reply:
really? Actually, it's not. they're all subtly different. I don't think this is a good idea.